python/README.md

```diff
@@ -26,57 +36,26 @@ optional arguments:
   -port PORT, --flightport PORT
                         Dremio flight server port. Defaults to 32010.
   -user USERNAME, --username USERNAME
-                        Dremio username. Defaults to "dremio".
+                        Dremio username.
   -pass PASSWORD, --password PASSWORD
```

Comment:
We should make it clear this is not applicable for DC connections (validate this), and not needed when using a PAT.

Reply:
By validate this do you mean check to see if the user is trying to connect to cloud and warn them if they're supplying username and password along with the token?

Reply:
Not at this point. Just make clear in the README